docs/automation-rules.rst

```diff
@@ -52,14 +52,18 @@ Currently, the following actions are available:
 
 - **Activate version**: Activates and builds the version.
 - **Hide version**: Hides the version. If the version is not active, activates it and builds the version.
-  See more in :ref:`versions:Version States`.
+  See :ref:`versions:Version States`.
+- **Make version public**: Sets the version's privacy level to public.
+  See :ref:`versions:privacy levels`.
+- **Make version private**: Sets the version's privacy level to private.
+  See :ref:`versions:privacy levels`.
```

Comment:
Shouldn't we note here that these are only available in "Read the Docs for Business"?

Reply:
I thought about that, the linked section already mentions that, so not sure.

Reply:
OK. Maybe not need to repeat that here, then.